### DISCUSSION TITLE
How to use the Flynt with the woocormmerce?

### DISCUSSION THREAD

**Discussion by forevereffort** [flyntwp/flynt#358] [Q&A] [answered]
Hello

since last year, I have been using the Flynt to develop the wordpress blog theme. It was a perfect. now I am going to develop the woo-theme with the Flynt. but I don't know how I begin it from sctracth.

is there any sample for Flynt && Woocormmerce? is there any issue?
Please let me know

Regards

**Comment by timohubois** [comment, marked as answer]
Hi @forevereffort I have also started to implement woocommerce into one project. But the project goes slowly forward, unfortunately it is for one customer who has been very limited due to lockdown.

I have started with a woocommerce.php at the root theme level and then I created twig-templates. Inside this templates I rendered custom woocommerce components.

To getting started you can take a look inside the timber documentation: https://timber.github.io/docs/guides/woocommerce/

Hope this helps.

**Reply by forevereffort** [reply]
Hi @timohubois 

Thanks for your reply. If you have a good progress, please let me know so I can learn from you.

Regards

**Reply by timohubois** [reply]
Sure, after the project will be done I planed to publish my way to implement woocommerce here – for now i can’t say how long it will take.

**Reply by timohubois** [reply]
What kind of shop to you plan to implement? I may be able to recommend an alternative to you. If you like to exchange, write me – but please outside of this discussion ;).

**Reply by forevereffort** [reply]
@forevereffort Good, How, Where can I contact to you?

**Reply by munts** [reply]
@timohubois and @forevereffort Did either of you ever get woocommerce fully integrated into a FlyntWP theme?  I have tried following the various tutorials online and I am still running into issues and was wondering you either of you got them all resolved and if you had any tips or tricks...  Thanks

**Reply by joshuabillington** [reply]
@timohubois @forevereffort @munts This guide is pretty good for integrating Timber with WooCommerce: https://github.com/mindkomm/timber-integration-woocommerce

**Reply by munts** [reply]
@joshuabillington have you had success adding WooCommerce support to FlyntWP?  I have followed both integration docs suggested in this thread and I have not been successful yet.  I have noticed that Flynt uses a directory called templates where the the above guide uses /views/...  It also extends base.twig which isn't used in Flynt.  Then the guide you suggest discusses the use of header.php and footer.php, but Flynt does use these files, instead they build the header and footer via the _document.twig file...

So, I have been trying to follow the guides and make the necessary changes as I find the differences, but seeing as I am still not successful, I must be missing something.

Thanks,
munts

**Reply by joshuabillington** [reply]
@munts I've had some success, I might be working on a package for it in the future. You can swap out /views for /templates and the extend _document.twig instead of base.twig. That should get you closer. I found integrating the package as an add on, wasn't; as successful as directly integrating.
